include/video.h

```diff
@@ -171,35 +167,26 @@ enum class PixelFormat : uint8_t {
 	// (msb)5R 6G 5B(lsb)
 	// Example:
 	// uint16_t pixel = (red << 11) | (green << 5) | (blue << 0)
-	//
-	// SDL Equivalent: SDL_PIXELFORMAT_RGB565
-	// FFmpeg Equivalent: AV_PIX_FMT_RGB565
 	RGB565_Packed16 = 16,
 
 	// 16.7M (24-bit) true colour, 8 bits per red/blue/green component;
-	// stored as packed 24-bit data
+	// stored as a sequence of three packed uint8_t values in BGR byte
+	// order, also known as memory order. This format is endian-agnostic.
 	//
-	// Stored as array of uint8_t in BGR memory order (endian agnostic)
 	// Example:
 	// uint8_t *pixels = image.image_data;
 	// pixels[0] = blue; pixels[1] = green; pixels[2] = red;
-	//
-	// SDL Equivalent: SDL_PIXELFORMAT_BGR24
-	// FFmpeg Equivalent: AV_PIX_FMT_BGR24
 	BGR24_ByteArray = 24,
 
-	// 16.7M (32-bit) true colour; 8 bits per red/blue/green component;
-	// stored as packed uint32 data with highest 8 bits unused
+	// Same as BGR24_ByteArray but padded to 32-bits. 16.7M true colour,
+	// 8 bits per red/blue/green/empty component; stored as a sequence of
+	// four packed uint8_t values in BGRX byte order, also known as
+	// memory order. This format is endian-agnostic.
 	//
-	// Stored as array of uint32_t in host native endianess.
-	// Each uint32_t is layed out as follows:
-	// (msb)8X 8R 8G 8B(lsb)
 	// Example:
-	// uint32_t pixel = (unused << 24) | (red << 16) | (green << 8) | (blue << 0)
-	//
-	// SDL Equivalent: SDL_PIXELFORMAT_XRGB8888
-	// FFmpeg Equivalent: AV_PIX_FMT_0RGB32
-	XRGB8888_Packed32 = 32
+	// uint8_t *pixels = image.image_data;
+	// pixels[0] = blue; pixels[1] = green; pixels[2] = red; pixels[3] = empty;
+	BGRX32_ByteArray = 32,
```

Comment:
Maybe leave the SDL/FFmpeg equivalents for the rest, just get rid of the SDL equivalent for `BGRX32_ByteArray`? The rest appear to be correct and could be helpful.

Reply:
ok! re-added those SDL/FFmpeg equivalents, but not mentioning any SDL/FFmpeg equivalents for `BGRX32_ByteArray`. It's kind of inconsistent, but I have no concerns either way! :sweat_smile:

Reply:
Well, we're listing equivalents where there are equivalents, and not listing them where there aren't 😄 

The FFmpeg equivalent for `BGRX32_ByteArray` is valid, you might wanna put that back too. The only thing "missing" is the the SDL equivalent for `BGRX32_ByteArray`. You could even say this has no SDL equivalent.

Reply:
added :)